### DISCUSSION TITLE
how to generate .geo file containing the physical line of boundaries

### DISCUSSION THREAD

**Discussion by damulaoshi** [neperfepx/neper#754] [Q&A] [answered]
Dear Romain,
First of all thank you so much for your software! 

I have a question when I use NEPER  generate some .geo file, then I need to use gmsh to do the mesh using some irregular quad elements. Is it possible in NEPER to define the physical lines of the boundaries? For example, something like this:
![image](https://github.com/neperfepx/neper/assets/75380898/8a0de835-0f28-4814-995e-550d3efbdf48)

I saw in .geo file, the physical surface has been defined. 

Here is the 2D code:
neper -T -n 20 -dim 2 -domain "square(1,1)" -morpho "centroidal" -format geo,tess -o 2Dcentroidal

Thank you before!
Best regards,
damulaoshi

**Comment by rquey** [comment, marked as answer]
Available in version 4.6.2-8.

**Reply by damulaoshi** [reply]
Dear Romain,
That is really good news! Thank you so much for answering this question!

Best regards
